packages/open-next/src/core/routing/util.ts

```diff
@@ -76,11 +76,9 @@ export function convertRes(res: OpenNextNodeResponse): InternalResult {
   // When using HEAD requests, it seems that flushHeaders is not called, not sure why
   // Probably some kind of race condition
   const headers = parseHeaders(res.getFixedHeaders());
-  const isBase64Encoded = isBinaryContentType(
-    Array.isArray(headers["content-type"])
-      ? headers["content-type"][0]
-      : headers["content-type"],
-  );
+  const isBase64Encoded =
+    isBinaryContentType(headers["content-type"]) ||
+    !!headers["content-encoding"];
```

Comment:
Just want to be sure this doesn't break something. In the past, the `content-type` somehow was sometimes an array. Was that fixed to always return a string now?

Reply:
Yeah it should be fine, `parseHeaders` is called just before and this function should ensure that all headers are string